docs/api.rst

```diff
@@ -3875,7 +3875,7 @@ Refreshing asset icons
 .. http:patch:: /api/(version)/assets/icon/modify
 
    Doing a PATCH on the asset icon endpoint will refresh the icon of the given asset.
-   First, the cache of the icon of the given asset is deleted and then requeried from CoinGecko and saved to the filesystem.
+   First, the cache of the icon of the given asset is deleted and then required from CoinGecko and saved to the filesystem.
```

Comment:
This is wrong. It's not required. It's re-queried.

```suggestion
   First, the cache of the icon of the given asset is deleted and then re-queried from CoinGecko and saved to the filesystem.
```

Reply:
done